packages/modules/viewer-react/src/tests/services/iModelViewerHelper.test.ts

```diff
@@ -0,0 +1,226 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Cartographic } from "@itwin/core-common";
+import type { BlankConnectionProps } from "@itwin/core-frontend";
+import { BlankConnection } from "@itwin/core-frontend";
+import { Range3d } from "@itwin/core-geometry";
+
+import * as iModelService from "../../services/iModel/IModelService";
+import {
+  createBlankConnection,
+  getConnectionType,
+  openConnection,
+} from "../../services/iModel/iModelViewerHelper";
+import type { RequiredViewerConnectionProps } from "../../types";
+import { ConnectionType } from "../../types";
+
+jest.mock("@itwin/core-frontend", () => {
```

Comment:
I really wanted to extract this and use it in the BaseInitializer test, but had issues with jest hoisting/initializing modules.